snippet-extractor-output/snippets.yaml

```diff
@@ -3358,6 +3378,26 @@
       range.valuesAsJson = [[error]];
       await context.sync();
     });
+'Excel.ErrorCode:enum':
```

Comment:
This should map to [CustomFunctions.ErrorCode](https://docs.microsoft.com/en-us/javascript/api/custom-functions-runtime/customfunctions.errorcode?view=excel-js-preview)

Reply:
Removing mapping per offline conversation.